.openshift-ci/crawler/init.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+# These commands will be executed on a freshly instantiated GCP VM to prepare
+# it for the following crawling and repackaging. The main purpose here is to
+# set up all the necessary variables needed on the next steps.
+
+export BASH_ENV=/tmp/bash_env
+touch $BASH_ENV
```

Comment:
[ultra-nit] Since this script is run in a retry loop, should we try force removing the `BASH_ENV` file? Same for the kernel-packer repository.
```bash
rm -f "$BASH_ENV"
rm -rf kernel-packer
touch "$BASH_ENV"
```